protos/orchestrator_service.proto

```diff
@@ -31,6 +31,7 @@ message ActivityResponse {
     int32 taskId = 2;
     google.protobuf.StringValue result = 3;
     TaskFailureDetails failureDetails = 4;
+    string completionToken = 5;
```

Comment:
I am a bit confused... how can `ActivityResponse` contain a completion token if the `ActivityRequest` does not? Or is this because you are reusing the data structure in a different context, where the completion token has been supplied by some other means than `ActivityRequest`?

If so then perhaps adding a line of comment (e.g. "only used when ...") would be helpful.

Reply:
The completion token comes from the work item message, which is a container for the `ActivityRequest`. See [here](https://github.com/microsoft/durabletask-protobuf/blob/88fa1b2081cbf19eddf446362afda0eb9690ce48/protos/orchestrator_service.proto#L614)